freqtrade/tests/optimize/test_backtesting.py

```diff
@@ -539,9 +539,11 @@ def test_backtest(default_conf, fee, mocker) -> None:
         # Check open trade rate alignes to open rate
         assert ln is not None
         assert round(ln.iloc[0]["open"], 6) == round(t["open_rate"], 6)
-        # check close trade rate alignes to close rate
+        # check close trade rate alignes to close rate or is between high and low
```

Comment:
I don't understand this one: "check close trade rate alignes to close rate or is between high and low" why ?

Reply:
after this PR, backtesting is not only using open or close price, but also ROI (%) / stoploss (%) - which is "somewhere" between high and low.

maybe we can also remove this assert statement as we're testing all of this in the detailed tests ... not sure about that though

Reply:
"close rate" should be either the "open rate" of the next candle or between low and high of the current one. that is what you mean ?